src/inverseRobot.js

```diff
@@ -24,7 +24,24 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const obj = {};
+  const keys = Object.keys(robot);
+
+  for (let i = 0; i <= keys.length; i++) {
+    for (let j = i + 1; j <= keys.length; j++) {
+      if (robot[keys[i]] === robot[keys[j]]) {
+        return null;
+      }
+    }
+  }
```

Comment:
```suggestion
  for (let i = 0; i <= keys.length; i++) {
    // 1) Check if your obj already has own property and return null if it's true
    // 2) Add new property to new robot
  }
```
You can rewrite this part without additional loop and perform adding new entries to your object in main loop. Google for `hasOwnProperty` method.
Also I recommend you to rename `obj` to something meaningful like `invertedRobot`.

Reply:
It's genius, really!!!